### PR TITLE
fix: better snapshot of highway test

### PR DESCRIPTION
**Pull request by ffMathy**
Makes the "highway test" more realistic by providing actual typing values.

### CODE REVIEW

**Comment by ffMathy** [comment]
Oh, this also fixes https://github.com/ftrackhq/ftrack-ts-schema-generator/issues/29.

**Comment by ffMathy** [comment]
And if you'd like to see the original change separated from the naming change, they are in individual commits. 

[af084ca](https://github.com/ftrackhq/ftrack-ts-schema-generator/pull/30/commits/af084ca85b1dfc954e768c0dd078c0321a105133)
[8251cb9](https://github.com/ftrackhq/ftrack-ts-schema-generator/pull/30/commits/8251cb970443b739b894c31aac4477c6d34778ca)

File: source/emit.ts [hunk 1]
```diff
@@ -149,8 +149,8 @@ export async function emitToString(
       ] as const;
     }
 
-    export type Type = ReturnType<typeof getTypes>[number];
-    export type TypeName = Type["name"];
+    export type TypeRuntimeInformation = ReturnType<typeof getTypes>[number];
```
Comment (gismya) [inline source/emit.ts:152]: I like runtime. But maybe have just `Runtime`as a prefix to them all? Shorter names, makes them easier to find "grouped" in the intellisense? It is it more important that they are grouped with the generic type?

Comment (ffMathy) [reply source/emit.ts:152]: That's totally fine for me as well.

Maybe you can make that correction if that's alright? I went on vacation now, so won't have time until January. I still hope this is a help in the right direction.

Comment (gismya) [reply source/emit.ts:152]: I'll do that. Happy holidays!

**Review by gismya — APPROVED** [review]

**Pull request merged**